From 060ce20b9896abbb7bd44f9637d78fbfd0ff1f23 Mon Sep 17 00:00:00 2001
From: Frank Wunderlich <frank-w@public-files.de>
Date: Sat, 5 Jul 2025 22:58:15 +0200
Subject: [PATCH 31/84] arm64: dts: mediatek: mt7988a-bpi-r4: drop readonly
 from bl2 partition

This flag must be removed to allow nand install scripts writing the bl2.

Fixes: 6b7642e9d095 ("arm64: dts: mediatek: mt7988a-bpi-r4: configure spi-nodes")
Signed-off-by: Frank Wunderlich <frank-w@public-files.de>
---
v8: new patch
---
 arch/arm64/boot/dts/mediatek/mt7988a-bananapi-bpi-r4.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/mediatek/mt7988a-bananapi-bpi-r4.dtsi b/arch/arm64/boot/dts/mediatek/mt7988a-bananapi-bpi-r4.dtsi
index 7c9df606f60d..6436263d2328 100644
--- a/arch/arm64/boot/dts/mediatek/mt7988a-bananapi-bpi-r4.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt7988a-bananapi-bpi-r4.dtsi
@@ -448,7 +448,6 @@ partitions {
 		partition@0 {
 			label = "bl2";
 			reg = <0x0 0x200000>;
-			read-only;
 		};
 	};
 };
-- 
2.30.2

